home.css

```diff
@@ -1,133 +1,280 @@
+    transform: translate(-50%, -50%); /* precise centering; see link below */
+}
+.main-title{
+    text-align: center;
+    font-family: 'Abel', sans-serif;
+    color: #FFF;
+    text-shadow: 2px 2px black;
+    transform: translateY(-40%);
+    border-radius:10px;
+    backdrop-filter: blur(20px);
+    font-size: 3.0rem;
+}
 
-.header-nav__list__item a:hover {
-    background-color:#fca311
+.middle-section{
+    padding: 3rem;
 }
-.header-nav__list__item a:active {
-    color: red;
+.titulo-about{
+    text-align: center;
+    color: #222831;
 }
-.header-nav__theme {
+.about-us-tittle{
+    text-shadow: 2px 2px white;
+    color: #222831;
+}
+.about-us{
     display: flex;
-    align-items: center;
+    flex-direction: row;
     justify-content: center;
-    flex-grow: 3;
+    padding-top: 4rem;
 }
-.header-nav__theme span {
-    padding: 10px;
+.first-profile-image{
+    width: 23rem;
+    box-shadow: 10px 10px 5px grey;
+}
+.primer-parrafo{
+    width: 40rem;
+    text-align: right;
+    font-family: 'Varela Round', sans-serif;
+    font-size: 1.2rem;
 }
 
-/*Styles for the presentation images using BEM*/
-
-.main-background {
-    position: relative;
-    width: 100%;
-    max-height: auto;
+.second-middle-section{
+    padding: 3rem;
 }
-.main-background__img {
-    position: relative;
-    width: 100%;
+.about-us-segundo{
     display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+.segundo-parrafo{
+    width: 40rem;
+    text-align: left;
+    font-family: 'Varela Round', sans-serif;
+    font-size: 1.2rem;
+}
+.second-profile-image{
+    width: 23rem;
+    box-shadow: 10px 10px 5px grey;
+}
+.third-section-tittle{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-top: 2rem;
 }
-.main-background__presentation {
+.third-section{
     display: flex;
+    flex-wrap: wrap;
     flex-direction: row;
-    width: 70%;
+    padding: 1rem;
+    justify-content: center;
+    justify-content: space-around;
 }
-#moving-img {
-    position: relative;
-    top: -150px;
-    left: 15%;
-}
-.main-background__presentation img {
-    width: 25%;
-    height: 345px;
-    object-fit: cover;
-    flex-grow: 1;
-}
-/*Styles for the presentation using BEM*/
-.main-presentation {
-    padding-left: 8vw;
-    padding-right: 8vw;
+.primer-tarjeta{
     display: flex;
     flex-direction: column;
-    align-items: center;
+    padding: 0.5rem;
+    background-color: #eeeeee;
+    box-shadow: 0 6px 8px 0 rgba(0,0,0,0.7);
+    transition: 0.3s;
+    margin: 0.7rem;
 }
-.main-presentation h1 {
-    font-family: sans-serif;
+.primer-tarjeta:hover{
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
 }
-.main-presentation__description p{
-    text-align: justify;
-    text-justify: inter-word;
+.primer-imagen-tarjeta{
+    width: 20rem
 }
-.main-presentation__projects {
+.contact-section-tittle{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-top: 2rem;
```

Comment:
Simplify with padding: (padding-top, padding-right, padding-bottom, padding-left).

Reply:
That would ruin responsive design, that's the reason I used that.